teamcity/tests/test_e2e.py

```diff
@@ -3,14 +3,26 @@
 # Licensed under a 3-clause BSD style license (see LICENSE)
 import pytest
 
-from .common import CONFIG
+from datadog_checks.base import AgentCheck
+from datadog_checks.teamcity.constants import SERVICE_CHECK_OPENMETRICS
 
+from .common import OPENMETRICS_INSTANCE, REST_INSTANCE, USE_OPENMETRICS
 
-# Minimal E2E testing
+
+@pytest.mark.skipif(USE_OPENMETRICS, reason="Not available in OpenMetricsV2 check")
 @pytest.mark.e2e
 def test_e2e(aggregator, dd_agent_check):
     # Prevent the integration from failing before even running the check
     with pytest.raises(Exception):
```

Comment:
Does anything speak against checking for a more exact exception type here?

Reply:
The check's exception type is `Exception`, but I can add the `match` parameter as you suggested in another comment to make the error matching more specific